apps/camera_pipe/camera_pipe_generator.cpp

```diff
@@ -10,7 +10,7 @@ using namespace Halide;
 using namespace Halide::ConciseCasts;
 
 // Shared variables
-Var x, y, c, yi, yo, yii, xi;
```

Comment:
Are the changes in this file necessary, or the result of intermediate states while debugging?

Reply:
Not strictly necessary, but I don't think they're hurting anything, either.

Reply:
I would prefer without then. I find "input" clearer than "func_in", and the explicit names are in theory redundant (at least on os x and linux, where we should be sniffing the dwarf syms. If we're not, that's a separate bug.)

Reply:
@abadams -- Okay, I will do that, but just so you know "input" shadows a class field and the names were _not_ being properly sniffed, which is why making them explicit worked around the name-clashing-after-reset bug.

Reply:
Done. The camera pipeline is now untouched by this PR.